src/main/java/application/MenuController.java

```diff
@@ -374,9 +375,11 @@ private NotificationMessageBean doSync(FormplayerSyncScreen screen, HqAuth auth)
             return new NotificationMessageBean("Session error, expected sync block but didn't get one.", true);
         }
         if(responseEntity.getStatusCode().is2xxSuccessful()){
-            return new NotificationMessageBean("Case claim successful", false);
+            CaseAPIs.forceRestore(restoreFactory);
```

Comment:
@wspride just checking that this is necessary? I thought that webapps already did this when a claim is successful.